bin/adhoc-scripts/checkStuckLQE.py

```diff
@@ -72,9 +72,9 @@ def commonDataLocation(element):
     return commonLoc
 
 def main():
-    if 'WMAGENT_CONFIG' not in os.environ:
-        os.environ['WMAGENT_CONFIG'] = '/data/srv/wmagent/current/config/wmagent/config.py'
-    config = loadConfigurationFile(os.environ["WMAGENT_CONFIG"])
+    if 'WMAGENT_CONFIG_FILE' not in os.environ:
```

Comment:
the correct environment variable name is: WMA_CONFIG_FILE, but I am sure this (WMAGENT_CONFIG) is one of the variables which is needed at other places as  well. I think in the `deploy/` scripts. We should remember to check those as well at some later stage.

Reply:
Thanks @todor-ivanov , silly mistake from my side. The new commit should fix this